src/views/Calendar.vue

```diff
@@ -183,6 +184,18 @@ export default {
 				})
 			}
 		}, 1000 * 60)
+		this.checkForUpdatesJob = setInterval(async() => {
+			const tokens = this.$route.params.tokens.split('-')
+			const updated = await this.$store.dispatch('getUpdatedPublicCalendars', { tokens })
+			updated.forEach(calendar => {
+				this.$store.dispatch('updateCalendar', { calendar })
+			})
+		}, 1000 * 20)
```

Comment:
Well, we were facing issues in using Nextcloud Calendar for schedule updates of live events. Both, the public calendar link and the embedded view were used to display the schedule.

We have many, many events in a very short period of time. Sometimes these events suddenly change, get other locations or are moved to another time slot. In this use case 20 seconds is sometimes even too long.

I hence definitely think we need a corresponding switch. Maybe a calendar-wide option like "Requires real time updates" might me useful?

Reply:
As this is not a push connection, but keeps querying your server every 20 s, this might create a lot of load (depending on the number of clients).

Reply:
I was suggesting to use a configuration setting, so that the instance admin would be able to pick a delay. You'll need a very strong setup to handle 20 seconds polling with many clients.

At some point what you want is to a separate websocket service to push changes, similar to https://github.com/nextcloud/notify_push (but that's probably not coming soon).

Reply:
I'm happy to create a setting for this - 1 Minute default works for me too. Would love to see something using `notify_push` instead. But one step at a time.

Not sure about the load this creates as if there are no changes it's a single query. But I guess they add up if many people leave their tabs open.